extensions/data-plane-selector/store/cosmos/data-plane-instance-store-cosmos/docs/schema.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS edc_data_plane_instance
+(
```

Comment:
I think this one is not needed for Cosmos

Reply:
Yes sure.